lib/core/src/lib/layout/components/layout-container/layout-container.component.scss

```diff
@@ -31,7 +32,7 @@ adf-layout-container {
     overflow: hidden;
 }
 
-/* stylelint-disable selector-class-pattern */
+/* stylelint-disable selector-class-pattern, declaration-no-important */
 #{$mat-sidenav-content},
 #{$mat-drawer-transition} #{$mat-drawer-content} {
```

Comment:
Do we need this rule?
Can't we get the same results without touching material selectors?